lib/SearchAndSort/tests/SearchAndSort-test.js

```diff
@@ -96,4 +97,11 @@ describe('SearchAndSort', () => {
       expect(onResetAllSpy.called).to.be.true;
     });
   });
+
+  describe('passing custom pane sub ', () => {
+    it('should display correct header', () => {
+      expect(searchAndSort.customPaneSubIsPresent).to.be.true;
```

Comment:
I think this test is excessive. If the title is present then we can assume that the component is present.